onnxruntime/core/providers/tensorrt/tensorrt_execution_provider_info.cc

```diff
@@ -122,10 +125,20 @@ TensorrtExecutionProviderInfo TensorrtExecutionProviderInfo::FromProviderOptions
           .AddAssignmentToReference(tensorrt::provider_option_names::kEpContextFilePath, info.ep_context_file_path)
           .AddAssignmentToReference(tensorrt::provider_option_names::kEpContextEmbedMode, info.ep_context_embed_mode)
           .AddAssignmentToReference(tensorrt::provider_option_names::kEngineHwCompatible, info.engine_hw_compatible)
+          .AddValueParser(
```

Comment:
i assume this function is similar to the one added for cuda stream for supporting enabling user_compute_stream Python API
https://github.com/microsoft/onnxruntime/pull/19229
https://github.com/microsoft/onnxruntime/pull/20168
We might enable onnx_bytestream option as well later.